frontend/src/scenes/plugins/pluginsLogic.ts

```diff
@@ -110,11 +111,13 @@ export const pluginsLogic = kea<
                     capturePluginEvent(`plugin source edited`, response)
                     return { ...plugins, [id]: response }
                 },
-                updatePlugin: async ({ id }) => {
+                updatePlugin: async ({ id, pluginChanges }) => {
                     const { plugins } = values
-                    const response = await api.update(`api/organizations/@current/plugins/${id}`, {})
+                    const response = await api.update(`api/organizations/@current/plugins/${id}`, pluginChanges)
                     capturePluginEvent(`plugin updated`, response)
-                    actions.pluginUpdated(id)
+                    if (!Object.keys(pluginChanges).length) {
+                        actions.pluginUpdated(id)
+                    }
```

Comment:
Small problem here. This endpoint by default fetches the latest version of the plugin. It, ughm, updates it, :). Sending `pluginChanges` here will make it a) update to the newest verison, b) update pluginChanges. I think we should split this into two actions. I definitely wouldn't expect a plugin to upgrade if I click the "is global" checkbox.